notebooks/microphysics.ipynb

```diff
@@ -90,7 +50,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install numpy scipy pandas matplotlib xarray scikit-learn tensorflow netcdf4 h5netcdf tqdm pyyaml s3fs pyarrow"
+    "!pip install numpy scipy pandas matplotlib xarray scikit-learn tensorflow netcdf4 h5netcdf tqdm pyyaml s3fs pyarrow mlmicrophysics"
```

Comment:
Can you add a second cell after this one that includes 
`! pip install --upgrade pandas`

Reply:
done!